haystack/pipeline.py

```diff
@@ -591,6 +592,70 @@ def run(self, **kwargs):
         return output
 
 
+class QuestionGenerationPipeline(BaseStandardPipeline):
```

Comment:
I understand that having standard pipelines, such as `QuestionGenerationPipeline`, `RetrieverQuestionGenerationPipeline`, and `QuestionAnswerGenerationPipeline`, make it easier to run a pipeline because there is no need to define the individual pipeline components. However, with more and more standard pipelines, how do users know which standard pipeline fits there needs? We already have `DocumentSearchPipeline`, `ExtractiveQAPipeline`, `GenerativeQAPipeline `, `FAQPipeline`, `SearchSummarizationPipeline`, and `TranslationWrapperPipeline`. Maybe we need an overview of all these pipelines in the documentation? And some guidelines when to define a new standard pipeline?